processor/src/test/java/org/mapstruct/ap/test/bugs/_1140/AirplaneWithNoAccessors.java

```diff
@@ -0,0 +1,6 @@
+package org.mapstruct.ap.test.bugs._1140;
```

Comment:
I wouldn't call this a bug. I think some other test package would be a better fit.